ci/container/container-run.sh

```diff
@@ -39,17 +39,17 @@ CTR=0
 while test $# -gt $CTR; do
     case "$1" in
         -h | --help) usage && exit 0 ;;
-        -f | --full) echo "The legacy image has been deprecated, --full is not an option anymore." && exit 0 ;;
+        -f | --full) echo "The legacy image has been deprecated, --full is not an option anymore." >&2 && exit 0 ;;
```

Comment:
How about we introduce:

```
log() {
  echo "$@" >&2
}
```
and use it everywhere where we need to echo to stderr?

Reply:
Good idea.  Changeset incoming, please stand by...

Reply:
This was resolved.